diff --git a/node_modules/@types/moment-timezone/moment-timezone.d.ts b/node_modules/@types/moment-timezone/moment-timezone.d.ts
index 7b24d8f..21c728c 100644
--- a/node_modules/@types/moment-timezone/moment-timezone.d.ts
+++ b/node_modules/@types/moment-timezone/moment-timezone.d.ts
@@ -62,6 +62,4 @@ declare module 'moment' {
     zoneAbbr(): string;
     zoneName(): string;
   }
-
-  const tz: MomentTimezone;
 }
